src/Identifier/Normalizer/DateTimeIdentifierDenormalizer.php

```diff
@@ -14,18 +14,25 @@
 namespace ApiPlatform\Core\Identifier\Normalizer;
 
 use ApiPlatform\Core\Exception\InvalidIdentifierException;
-use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+use Symfony\Component\Serializer\Exception\NotNormalizableValueException;
 use Symfony\Component\Serializer\Normalizer\DateTimeNormalizer;
-use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
 
-final class DateTimeIdentifierDenormalizer extends DateTimeNormalizer implements DenormalizerInterface
+final class DateTimeIdentifierDenormalizer extends DateTimeNormalizer
 {
     public function denormalize($data, $class, $format = null, array $context = [])
     {
         try {
             return parent::denormalize($data, $class, $format, $context);
-        } catch (InvalidArgumentException $e) {
+        } catch (NotNormalizableValueException $e) {
```

Comment:
@soyuka, I found this bug while adding an unit test. Can you check it is correct please?

Reply:
Oh yes I used the `normalize` signature to find the exception. This is indeed correct.